sdcm/gemini_thread.py

```diff
@@ -60,28 +60,38 @@ def run(self):
 
 class GeminiStressThread():  # pylint: disable=too-many-instance-attributes
 
+    gemini_result_file_name = "gemini_result.log"
```

Comment:
you never run gemini multiple times on the same loader ?
I would add some more ids into the same, same as the log.

Reply:
Added uuid to resultfile